set_ar5search_env.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Usage: source ./set_ar5search_env.sh
```

Comment:
We might want to use an `env.production` file for this.
See https://nextjs.org/docs/basic-features/environment-variables#environment-variable-load-order.